docs/UserGuide.adoc

```diff
@@ -369,43 +369,60 @@ That's it for this section. Congratulations, you now know how to use the `find`
-kidney matching tests on a specific patient, refer to <<Matching a specific patient>>.
+You can either run a kidney matching test on a specific patient or on all patients in ORGANice.
+To learn how to run kidney matching tests on a specific patient,
+refer to <<Matching a specific patient>>.
+To learn how to run kidney matching tests on all patients,
+refer to <<Matching all patients>>.
 
 ==== Matching a specific patient
-This section will explain how to run kidney matching tests on a specific patient and run you through an example.
-To run a match of a specific patient, type in a command of the format `match ic/**NRIC**`.
+This section will explain how to run kidney matching tests on a specific patient and give you an example.
+To get a list of potential donors for a patient,
+type in a command of the format `match ic/**PATIENT NRIC**`.
 
-Here is an example. Suppose you want to match a specific patient called "John Doe" and his NRIC is "S1234568R". This is
-what you can do:
+Here is an example. Suppose you want to find potential donors for a patient called "John Doe" and his NRIC is "S1234568R".
+You will have to type `match ic/S1234568R` in the Command Box and press kbd:[Enter].
 
-. Type `match ic/S1234568R` in the Command Box and press kbd:[Enter].
+You will see a list of potential donors for John Doe along with the success
+rate of each donor, similar to Figure 13.
 
-. The list of donors and their success rate are displayed as shown in the screenshot below. +
-+
+//TODO need to change this to another screenshot
+.List of Donors that match John Doe
 image::MatchingDonorsDisplayed.png[width=790]
-+
+
 For more information on the success rate, you may refer to <<FAQ>>.
 
 ==== Matching all patients
 This section will explain how to run kidney matching tests on all patients stored in ORGANice and run you
 through an example.
-To match all the patients in ORGANice, do the following:
+To match all the patients in ORGANice,
+type `match ic/all` in the Command Box and press kbd:[Enter].
 
-. Type `match ic/all` in the command box and press kbd:[Enter].
+You will see a list of all the patients in ORGANice along with the number of
+potential donors for each patient.
 
-. The list of patients and the number of matching donors each patient have are displayed in the screenshot below.
-+
+//TODO change the Figure name as more diagrams are added.
+For example, you have three patients named 'Someone edited',
+'Gary Tennant' and 'Elle Brady' respectively stored in ORGANice.
+Some donors are also stored in ORGANice.
+If you type `match ic/all` in the Command Box and press kbd:[Enter],
+you will see 'Someone edited', 'Gary Tennant'
+and 'Elle Brady' along with the number of matching donors each patient has. +
+
+Referring to Figure 14, 'Someone edited' is displayed with the red text 'Matches: 3'.
+This means that 'Someone edited' has 3 potential donors.
```

Comment:
I would prefer this format: "Fig 14 below illustrates (idea)..." [photo] "As seen in fig 14, (explain)..."